frontend/app/layout/SideMenu.tsx

```diff
@@ -169,66 +160,83 @@ function SideMenu(props: Props) {
               <>
                 {index > 0 && <Divider style={{ margin: '6px 0' }} />}
 
-                  {category.items.filter((item: any) => !item.hidden).map((item: any) => {
-                    const isActive = isMenuItemActive(item.key);
-
-                    if (item.key === MENU.EXIT) {
-                      return (
-                        <Menu.Item
-                          key={item.key}
-                          style={{ paddingLeft: '20px' }}
-                          icon={<Icon name={item.icon} size={16} color={isActive ? 'teal' : ''} />}
-                          className={cn('!rounded-lg hover-fill-teal')}
-                        >
-                          {item.label}
-                        </Menu.Item>
-                      );
-                    }
+                {category.items.filter((item: any) => !item.hidden).map((item: any) => {
```

Comment:
what is going on in this file?

Reply:
I've included BETA badge to Spots menu item.